File: wallendar-app/app/src/main/java/com/td/wallendar/addmembers/MembersAdapter.java
package com.td.wallendar.addmembers;

import android.annotation.SuppressLint;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.td.wallendar.R;

import java.util.ArrayList;
import java.util.List;

public class MembersAdapter extends RecyclerView.Adapter<MembersViewHolder> {

    private List<String> members;

    public MembersAdapter(List<String> members) {
        this.members = members;
    }

    @NonNull
    @Override
    public MembersViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
        final View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.row_member, parent, false);
        return new MembersViewHolder(view);
    }

    @Override
    public void onBindViewHolder(@NonNull MembersViewHolder holder, int position) {
        System.out.println("BINDEANDO");
        holder.bind(members.get(position));
    }

    @Override
    public int getItemCount() {
        return members.size();
    }

    @SuppressLint("NotifyDataSetChanged")
    public void addToDataset(String member) {
        this.members.add(member);
        notifyItemInserted(members.size() - 1);
    }
}

File: wallendar-app/app/src/main/java/com/td/wallendar/addmembers/MembersAdapter.kt
package com.td.wallendar.addmembers

import android.annotation.SuppressLint
import android.view.LayoutInflater
import android.view.ViewGroup
import androidx.recyclerview.widget.RecyclerView
import com.td.wallendar.R

class MembersAdapter(private val members: MutableList<String>) : RecyclerView.Adapter<MembersViewHolder?>() {
    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MembersViewHolder {
        val view = LayoutInflater.from(parent.context).inflate(R.layout.row_member, parent, false)
        return MembersViewHolder(view)
    }

    override fun onBindViewHolder(holder: MembersViewHolder, position: Int) {
        holder.bind(members[position])
    }

    override fun getItemCount(): Int {
        return members.size
    }

    @SuppressLint("NotifyDataSetChanged")
    fun addToDataset(member: String) {
        members.add(member)
        notifyItemInserted(members.size - 1)
    }
}